octobot_trading/exchanges/parser/orders_parser_generic_ccxt.py

```diff
@@ -0,0 +1,296 @@
+import octobot_trading.constants as constants
+from octobot_trading.enums import (
+    ExchangeConstantsOrderColumns as OrderCols,
+    ExchangeOrderCCXTColumns as CCXTOrderCols,
+    OrderStatus,
+    TradeOrderSide,
+)
+from octobot_trading.enums import TradeOrderType as TradeOrderType
+import octobot_trading.exchanges.parser.orders_parser_ccxt as orders_parser_ccxt
+from octobot_trading.exchanges.parser.orders_parser_cryptofeed import (
+    CryptoFeedOrdersParser,
+)
+
+
+class GenericCCXTOrdersParser(orders_parser_ccxt.CCXTOrdersParser):
+    """
+    Dont override this class, use CCXTOrdersParser instead
+
+    always/only include bulletproof custom code into the parser to improve generic support
+
+    parser usage:   parser = GenericCCXTOrdersParser(exchange)
+                    orders = await parser.parse_orders(raw_orders)
+                    order = await parser.parse_order(raw_order)
+    """
+
```

Comment:
I don't think we want a generic one. In my opinion we only want **one** ccxt parser that we can override to adapt.
Before officially supporting we have to do some specific tests, we can't just try it with generic parsers and say it's supported in my opinion.

Reply:
Generic is not for the supported ones.
only the others you can select already, but aren't working well and aren't officially supported.
With the generic one, they have a way higher chance of success.

Also everything you think is not safe can be optional and turned off by default and later be an option in the coming experimental settings

But if you really don't want generic parser, it can be optional in the experimental settings, that will follow later

Reply:
Once again, it's a good idea but that's not the philosophy we have today. For now we want to add specific tentacle to support each exchange specifically. We want to be sure that we either fully support an exchange or not. We don't want to announce that we may support some part of an exchange with a generic algo which could produce unexpected effects if the exchange changes its logic.

Reply:
what about the following:
The generic parser will never be used.
Except if it's enabled in the hidden experimental settings (which will follow later). Mainly for us to test existing fixes faster